plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/expression/RewriteOr.java

```diff
@@ -60,6 +60,6 @@ public Optional<String> rewrite(Call call, Captures captures, RewriteContext<Str
         }
 
         return Optional.of(terms.stream()
-                .collect(Collectors.joining(") OR (", "(", ")")));
+                .collect(Collectors.joining(") OR (", "((", "))")));
```

Comment:
I wouldn't add braces here.
Whenever we combine expressions, it's the combining code to add braces if necessary.
that's why we add braces here
that's also why we add braces  eg 
 here https://github.com/trinodb/trino/blob/76bb02c759cdf95a7db1c65cd99fedb3e1d2b39d/plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/expression/GenericRewrite.java#L76

to avoid "over-bracing", let's add braces in `io.trino.plugin.jdbc.BaseJdbcClient#getAdditionalPredicate`

Reply:
Sure. That was my initial attempt so I'm fine with either of that approaches